examples/hitl/pick_throw_vr/config/experiment/headless_server.yaml

```diff
@@ -10,4 +10,5 @@ habitat_hitl:
       #   num_frames_to_save: 50
       #   filepath_base: "./debug_video"
       # exit_after: 40
-  remote_gui_mode: True
+  remote_gui:
```

Comment:
I have another change coming soon that adds the following:
```
  
  networking:
    # When enabled, a HITL app behaves as a server that takes input from a remote client. See pick_throw_vr example app.
    enable: False

    # We'll listen for incoming client connections at this port.
    port: 8888

    # Used with AWS elastic load balancer (ELB) health checks. Return a code based on whether the server is available to accept new client connections. Note the current server implementation only handles one client at a time.
    http_availability_server:
      enable: False
      port: 8889
      # HTTP codes
      code_available: 200  # "Ok"
      code_unavailable: 503  # "Service Unavailable"  
      ```

Reply:
Can you reword to match my naming? I.e. `networking.enable` instead of `remote_gui.enabled`.

Reply:
Sounds good!